components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -119,9 +318,37 @@ export default class MarketplaceItem extends React.Component {
             pluginIcon = <PluginIcon className='icon__plugin icon__plugin--background'/>;
         }
 
+        let localTag;
+        if (!this.props.downloadUrl) {
+            const localTooltip = (
+                <Tooltip id='plugin-marketplace__local-tooltop'>
+                    <FormattedMessage
+                        id='marketplace_modal.list.local.tooltip'
+                        defaultMessage='This plugin is not listed in the marketplace but was installed manually.'
```

Comment:
Agreed that you can manually install a plugin that's listed in the marketplace. To capture that more explicitly, I might swap `but` with `and`:
> This plugin is not listed in the marketplace and was installed manually.

Which is now true, but somehow sounds awkward to me in English. Do you feel the original phrase will be confusing in any context in which it will actually appear (vs. being interpreted as a generally true statement about installation method)?

I'd hesitate on using `server instance`, since that gets ambiguous in the context of high-availability clusters, but could do something like "This plugin is only installed in your Mattermost Server." But that's perhaps even more confusing, since the other plugins are also in various stages of being installed.

Reply:
What I believe that is confusing is the phrase:

> This plugin is not listed in the marketplace ... 

where we are referring to `marketplace` as the Marketplace Service and not this screen 'Plugins Marketplace` where you are seeing it listed. 

But that's just my two cents here and from an Admin perspective it's clear enough.